### PR TITLE
Note on privacy.

### PR DESCRIPTION
**Pull request by chris-wood**

### CODE REVIEW

File: draft-ietf-taps-transport-security.md [hunk 1]
```diff
@@ -1038,8 +1038,18 @@ analysis may compromise any protocol discussed in this survey. Applications usin
 Security Interfaces SHOULD take such limitations into consideration when using a particular
 protocol implementation.
 
+# Privacy Considerations
+
+Privacy enhancing (or offending) features are intentionally omitted from this survey.
+All protocols surveyed generally improve privacy by reducing information leakage via
```
Comment (tfpauly) [inline draft-ietf-taps-transport-security.md]: `All security protocols`, yes?

File: draft-ietf-taps-transport-security.md [hunk 2]
```diff
@@ -1038,8 +1038,18 @@ analysis may compromise any protocol discussed in this survey. Applications usin
 Security Interfaces SHOULD take such limitations into consideration when using a particular
 protocol implementation.
 
+# Privacy Considerations
+
+Privacy enhancing (or offending) features are intentionally omitted from this survey.
```
Comment (tfpauly) [inline draft-ietf-taps-transport-security.md]: Perhaps, `Analysis of how features improve or degrade privacy is intentionally omitted from this survey.`

**Review by tfpauly — CHANGES_REQUESTED** [review]

File: draft-ietf-taps-transport-security.md (hunk 1 above)
Comment (chris-wood) [reply draft-ietf-taps-transport-security.md]: Yep, fixed!

File: draft-ietf-taps-transport-security.md (hunk 2 above)
Comment (chris-wood) [reply draft-ietf-taps-transport-security.md]: WFM!

**Review by tfpauly — APPROVED** [review]

**Pull request merged**